Field_Simulation/html/example.html

```diff
@@ -0,0 +1,92 @@
+<html>
+<head></head>
+<body>
+	<canvas id='bitmap' width ='400' height = '400'></canvas>
+	<script>
+		var time;
+		var CIRCLE_TIME = 1 * 60 * 200;
+		var CIRCLE_RADIUS = 30;
+		var FIELD_SIZE = 30;
+		var mutex = false;
+		start();
+		//setInterval(loop,0.1);
+		//setTimeout(clear_lines,0.1);
+		function start()
+		{
+			time = new Date().getTime();
+		}
+		function loop(context)
+		{
+			var element = document.getElementById("bitmap");
+			var bitmap = element.getContext("2d");
+			var angle = 360 * get_ratio();
+			angle = radians(angle);
+			var circle = [Math.sin(angle)*CIRCLE_RADIUS,Math.cos(angle)*CIRCLE_RADIUS];
+			var zero = [200,200];
+			var vector = circular_field(angle);
+
+			circle[0] += zero[0]
+			circle[1] = (circle[1]+zero[1]);
+			vector[0] += circle[0];
+			vector[1] = (vector[1]+circle[1]);
+			
+			while(mutex);
```

Comment:
NO! This is Javascript, we don't block our only thread... What have you tried to achieve here?